Calculator/Calculator/AbsoluteLayout.xaml

```diff
@@ -11,46 +12,48 @@
                    AbsoluteLayout.LayoutFlags="All">
                 <Label AbsoluteLayout.LayoutBounds="0, 0, AutoSize, AutoSize"
                        AbsoluteLayout.LayoutFlags="PositionProportional" 
-                       x:Name="ExpressionText" Text="dymdtymdtum" 
+                       x:Name="ExpressionText" Text="" 
                        FontSize="Large" />
             </Frame>
             <AbsoluteLayout AbsoluteLayout.LayoutBounds="0, .18, 1, .2" 
                             AbsoluteLayout.LayoutFlags="All">
-                <Button Text="C" AbsoluteLayout.LayoutBounds="0, 0, AutoSize, AutoSize" AbsoluteLayout.LayoutFlags="PositionProportional"/>
-                <Button Text="/" AbsoluteLayout.LayoutBounds=".33, 0, AutoSize, AutoSize" AbsoluteLayout.LayoutFlags="PositionProportional"/>
-                <Button Text="X" AbsoluteLayout.LayoutBounds=".66, 0, AutoSize, AutoSize" AbsoluteLayout.LayoutFlags="PositionProportional"/>
-                <Button Text="Back" AbsoluteLayout.LayoutBounds=".99, 0, AutoSize, AutoSize" AbsoluteLayout.LayoutFlags="PositionProportional"/>
+                <Button FontSize="Large" Clicked="OperationHandler" Text="C" AbsoluteLayout.LayoutBounds="0, 0, AutoSize, AutoSize" AbsoluteLayout.LayoutFlags="PositionProportional"/>
```

Comment:
let's start each parameter with new line, so your Button will look like:
<Button FontSize="Large"
              Clicked="OperationHandler"
              Text="C"
              AbsoluteLayout.LayoutBounds="0, 0, AutoSize, AutoSize" 
              AbsoluteLayout.LayoutFlags="PositionProportional"/>

It's much more readable.